portal-test-integration/bnd.bnd

```diff
@@ -13,7 +13,10 @@ Export-Package:\
 	com.liferay.portlet.documentlibrary.util.test,\
 	com.liferay.portlet.notifications.test,\
 	com.liferay.portlet.social.test
-Import-Package: *;resolution:=optional
+Import-Package:\
```

Comment:
@dantewang @ChiZelin 
With this, we are still depending on portal-impl.
I think if you retrieve the class from portal class loader instead of using the default class loader, you will not need this.

Please resend.

Tina.